chia/full_node/weight_proof_v2.py

```diff
@@ -0,0 +1,1758 @@
+import asyncio
+import dataclasses
+import logging
+import math
+import random
+from concurrent.futures import as_completed
+from concurrent.futures.process import ProcessPoolExecutor
+from typing import Dict, List, Optional, Tuple, Union, Any
+
+from chia.consensus.block_header_validation import validate_finished_header_block
+from chia.consensus.block_record import BlockRecord
+from chia.consensus.blockchain_interface import BlockchainInterface
+from chia.consensus.constants import ConsensusConstants
+from chia.consensus.deficit import calculate_deficit
+from chia.consensus.full_block_to_block_record import header_block_to_sub_block_record
+from chia.consensus.pot_iterations import calculate_iterations_quality, calculate_sp_iters, is_overflow_block
+from chia.consensus.vdf_info_computation import get_signage_point_vdf_info
+from chia.types.blockchain_format.classgroup import ClassgroupElement, B
+from chia.types.blockchain_format.sized_bytes import bytes32
+from chia.types.blockchain_format.slots import ChallengeChainSubSlot, RewardChainSubSlot, ChallengeBlockInfo
+from chia.types.blockchain_format.sub_epoch_summary import SubEpochSummary
+from chia.types.blockchain_format.vdf import (
+    VDFInfo,
+    compress_output,
+    verify_compressed_vdf,
+)
+from chia.types.end_of_slot_bundle import EndOfSubSlotBundle
+from chia.types.header_block import HeaderBlock
+from chia.types.weight_proof import (
+    RecentChainData,
+    SubEpochChallengeSegmentV2,
+    SubEpochData,
+    SubEpochSegmentsV2,
+    SubSlotDataV2,
+    WeightProofV2,
+)
+from chia.util.block_cache import BlockCache
+from chia.util.hash import std_hash
+from chia.util.ints import uint8, uint32, uint64, uint128
+from chia.util.streamable import recurse_jsonify, dataclass_from_dict
+
+log = logging.getLogger(__name__)
+
+
+class WeightProofHandlerV2:
+    LAMBDA_L = 100
```

Comment:
Can we add a link to the flyclient paper and or explain where these values come from?